bin/bootstrap_practice_exercise.exs

```diff
@@ -0,0 +1,283 @@
+  def print_expected(expected, false), do: inspect(expected)
+
+  def print_test_case(
+        %{"description" => description, "cases" => sub_cases} = category,
+        properties,
+        module
+      ) do
+    """
+    describe \"#{description}\" do
+    #{Generate.print_comments(category, ["description", "cases"])}
+    #{Enum.map_join(sub_cases, "\n\n", &Generate.print_test_case(&1, properties, module))}
+    end
+    """
+  end
+
+  def print_test_case(
+        %{
+          "description" => description,
+          "property" => property,
+          "input" => input,
+          "expected" => expected
+        } = test,
+        properties,
+        module
+      ) do
+    %{name: name, variables: variables, error: error} = properties[property]
+    variable_list = Enum.map_join(variables, ", ", fn {var, _} -> var end)
+
+    """
+    @tag :pending
+    test \"#{description}\" do
+      #{Generate.print_comments(test, ["description", "property", "input", "expected", "uuid"])}
+      #{print_input(input)}
+      output = #{module}.#{name}(#{variable_list})
+      expected = #{print_expected(expected, error)}
+
+      assert output == expected
+    end
+    """
+  end
+end
+
+Mix.install([
+  {:jason, "~> 1.2"},
+  {:toml, "~> 0.6"}
+])
+
+[exercise] = System.argv()
+
+exercise_snake_case = String.replace(exercise, "-", "_")
+
+module =
+  exercise
+  |> String.split("-")
+  |> Enum.map_join("", &String.capitalize/1)
+
+## Step 1: create folder structure
+
+Mix.Generator.create_directory("exercises/practice/#{exercise}")
+Mix.Generator.create_directory("exercises/practice/#{exercise}/.docs")
+Mix.Generator.create_directory("exercises/practice/#{exercise}/.meta")
+Mix.Generator.create_directory("exercises/practice/#{exercise}/lib")
+Mix.Generator.create_directory("exercises/practice/#{exercise}/test")
+
+## Step 2: add common files
+
+# .formatter.exs
+format = """
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+]
+"""
+
+Mix.Generator.create_file("exercises/practice/#{exercise}/.formatter.exs", format)
+
+# mix.exs
+mix = """
+defmodule #{module}.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :#{exercise_snake_case},
+      version: "0.1.0",
+      # elixir: "~> 1.8",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      # {:dep_from_hexpm, "~> 0.3.0"},
+      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
+    ]
+  end
+end
+"""
+
+Mix.Generator.create_file("exercises/practice/#{exercise}/mix.exs", mix)
+
+# test/test_helper.exs
+test_helper = """
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+"""
```

Comment:
Minor, same indent as mentioned above

Reply:
Here the indentation looks good already 🤔 meaning: level 0, there's no indentation.